src/jit/morph.cpp

```diff
@@ -8226,7 +8226,7 @@ GenTree* Compiler::fgMorphCall(GenTreeCall* call)
     if (fgGlobalMorph && call->IsUnmanaged() && call->IsSuppressGCTransition())
     {
         // Insert a GC poll.
-        bool insertedBB = fgCreateGCPoll(GCPOLL_CALL, compCurBB);
+        bool insertedBB = fgCreateGCPoll(GCPOLL_CALL, compCurBB, compCurStmt);
```

Comment:
what about another call site:
https://github.com/dotnet/coreclr/blob/7f7f0e54713a6a7fcf3d2bdc7a15caa58eaec5f6/src/jit/morph.cpp#L7331-L7332

it also has a current statement, should it use it?

Reply:
That is legacy code - see https://github.com/dotnet/coreclr/issues/27186#issuecomment-542386711 for previous `GC_POLL` usage.